infra/deploy-aio.sh

```diff
@@ -1,45 +1,56 @@
 #! /bin/bash
 
 export CLUSTER_NAME=$1
-export RESOURCE_GROUP=$2
+export STORAGE_ACCOUNT_NAME=$2
+export SCHEMA_REGISTRY_NAME=$3
+export RESOURCE_GROUP=$4
 export SUBSCRIPTION_ID=$(az account show --query id -o tsv)
-export LOCATION=$3
+export LOCATION=$5
 
 # register providers
 az provider register -n "Microsoft.ExtendedLocation"
 az provider register -n "Microsoft.Kubernetes"
 az provider register -n "Microsoft.KubernetesConfiguration"
-az provider register -n "Microsoft.IoTOperationsOrchestrator"
 az provider register -n "Microsoft.IoTOperations"
 az provider register -n "Microsoft.DeviceRegistry"
 
 # install CLI extensions
 echo "Installing CLI extensions..."
-az extension add --name connectedk8s
-az extension add --name azure-iot-ops
+curl -L -o connectedk8s-1.10.0-py2.py3-none-any.whl https://github.com/AzureArcForKubernetes/azure-cli-extensions/raw/refs/heads/connectedk8s/public/cli-extensions/connectedk8s-1.10.0-py2.py3-none-any.whl   
+az extension add --upgrade --source connectedk8s-1.10.0-py2.py3-none-any.whl --yes
+az extension add --upgrade --name azure-iot-ops --yes
```

Comment:
For Azure IoT Ops compat with version 0.7.0, it would make sense to pin that version. Otherwise if a new version is released and the code is run then you could get an incompatible version deployed and break further script sections.

Reply:
And forgot to mention I would also suggest adding a WARNING block in the Readme that this repo is compatible with version Preview 0.7.0 of AIO.

Reply:
totally makes sense